### PR TITLE
pull request to merge branch hello-microverse to main 

### PR DESCRIPTION
**Pull request by Hadi111jafari**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+test.md
```
Comment (thecodechaser) [inline .gitignore:1]: - [ ]  Please add `node_modules` to your `.gitignore` file.

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## Hi @Hadi111jafari 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._
- [ ] Please add a descriptive README file to your project, Feel free to use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.
- [ ] Pull Request should have a descriptive title and description telling you what changes are inside. Please update the title and add a description to the Pull Request.
- [ ] Please ensure that your branch name should be descriptive (e.g.: hello-microverse), so that others can see what is being worked on. Right now your branch is called a feature, Please rename it. To rename your branch [Click here](https://github.com/Hadi111jafari/Hello-world/branches)

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Hadi111jafari** [comment]
I have a problem. When I rename the branch `feature` to `Hello-Microverse`, I can't reopen the pull request because it says that the branch `feature` has been deleted.
I think we have two options here. Either to create a new pull request or reopen this pull request with the `feature` branch.
what is your recommendation?

**Comment by thecodechaser** [comment]
@Hadi111jafari it is always a good practice to name the branch name descriptive, i would suggest you create a new branch with all the changes required and please also look at the changes i requested. provide description to your pull request and also use the readme file, You can use the template i mentioned in the review

**Comment by Hadi111jafari** [comment]
I did everything you mentioned here but for new branch and all the changes
I made I need to create new pull request and reviewing again is that OK ?
Should I create new pull request?

**Comment by thecodechaser** [comment]
@Hadi111jafari Yes you should creat new PR and in the description please link this PR as well so the next code Reviewer can see what changes i requested

**Comment by Hadi111jafari** [comment]
Ok thank you I'll do that but how do link this pull request in description ?

**Comment by thecodechaser** [comment]
@Hadi111jafari just paste the link of this PR and add text please read this PR before review.. Good luck

**Comment by Hadi111jafari** [comment]
Ok thanks in advance

**Pull request closed without merging**